cmd/evm/main.go

```diff
@@ -174,17 +176,23 @@ type VMEnv struct {
 	Gas   *big.Int
 	time  *big.Int
 	logs  []vm.StructLog
+
+	evm *vm.EVM
 }
 
-func NewEnv(state *state.StateDB, transactor common.Address, value *big.Int) *VMEnv {
-	return &VMEnv{
+func NewEnv(state *state.StateDB, transactor common.Address, value *big.Int, cfg *vm.Config) *VMEnv {
+	params.HomesteadBlock = new(big.Int)
```

Comment:
Is it wise to reset the HomesteadBlock here? Any code calling NewEnv post startup would immediately blow up consensus.